src/media-groups.js

```diff
@@ -736,13 +736,10 @@ export const setupMediaGroups = (settings) => {
   // DO NOT enable the default subtitle or caption track.
   // DO enable the default audio track
   const audioGroup = mediaTypes.AUDIO.activeGroup();
+  const groupId = (audioGroup.filter(group => group.default)[0] || audioGroup[0]).id;
 
-  if (audioGroup) {
```

Comment:
Is there a reason to remove this check? It doesn't seem to have much of an impact since it's on setup, but it ends up calling resetEverything in the main segment loader for video only streams.

Reply:
Good catch, going to revert this.